src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -2697,6 +2698,10 @@ private String launchCommandIn(ArgumentListBuilder args, File workDir, EnvVars e
 
             return stdout;
         } catch (GitException | InterruptedException e) {
+            if (e.getMessage().contains("unsupported option \"accept-new\"")) {
+                listener.getLogger().println(HyperlinkNote.encodeTo("https://plugins.jenkins.io/git-client/#plugin-content-ssh-host-key-verification/",
```

Comment:
Remove trailing slash from anchor in page
```suggestion
                listener.getLogger().println(HyperlinkNote.encodeTo("https://plugins.jenkins.io/git-client/#plugin-content-ssh-host-key-verification",
```